crates/vm/src/system/phantom/execution.rs

```diff
@@ -98,31 +103,37 @@ unsafe fn execute_e12_impl<F: PrimeField32, CTX: ExecutionCtxTrait>(
         vm_state.exit_code = Err(e);
         return;
     }
-    vm_state.pc += DEFAULT_PC_STEP;
-    vm_state.instret += 1;
+    *pc += DEFAULT_PC_STEP;
```

Comment:
nit: perhaps it'd be nice to have an `inst_inc!` or inst_advance!` macro to make these changes more uniform for later

Reply:
I'll keep this for later when i try to get rid of pc